Makefile

```diff
@@ -1,18 +1,13 @@
-ifeq ("$(GOPATH)","")
-$(error GOPATH must be set)
-endif
+export GO111MODULE=on
+MODULE = github.com/envoyproxy/ratelimit
 
 SHELL := /bin/bash
-GOREPO := ${GOPATH}/src/github.com/lyft/ratelimit
+GOREPO := ${GOPATH}/src/github.com/envoyproxy/ratelimit
```

Comment:
looks like this is not used anymore in the rest of the file. Not sure if this is needed for the internal Lyft build.

Reply:
This isn't used anywhere else in the makefile we can delete it.